src/Widgets/calendar/GridDay.vala

```diff
@@ -56,21 +57,30 @@ public class DateTime.Widgets.GridDay : Gtk.EventBox {
 
         label = new Gtk.Label (null);
 
+        event_dots = new Gee.HashMap<string, Gtk.Widget> ();
+
+        model.events_added.connect (add_event_dots);
+        model.events_removed.connect (remove_event_dots);
+
         event_grid = new Gtk.Grid ();
         event_grid.halign = Gtk.Align.CENTER;
         event_grid.height_request = 6;
 
-        var grid = new Gtk.Grid ();
```

Comment:
It doesn't look like you're accessing this grid from outside of the `construct` block. Why did you make it available to the whole class?

Reply:
So I could use it on the add_event_dots method.